ccd-definition/CaseEvent/Camunda/NotificationEventsLRspec.json

```diff
@@ -219,5 +219,18 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
```

Comment:
Can this be moved to a CUI related file please. So that we can toggle the feature on and off